beeswax/logs/ad_log.proto

```diff
@@ -37,9 +37,9 @@ message RequestLogMessage {
 // Next Tag: 17
 // ImpressionLogMessage is created for every won auction.
 message ImpressionLogMessage {
+  // NOTE: auctionid is an opaque unique id. Client cannot rely on the meaning
+  // of elements within it or the format of the id.
   optional base.EventId auctionid = 1;
-  // The unique identifier of the auction per buzz_key.
-  // Format: <auctionid.timestamp>.<auctionid.hostid>.<auctionid.tid>.<buzz_key>
   optional string auctionid_str = 12;
```

Comment:
I don't think this fully matches the definition in our codebase. We're missing `ImpressionLogMessage.requestid_str` at a minimum. https://github.com/BeeswaxIO/beeswaxio/blob/8f23b17417eb59f8cff1329e69e2a8f211aff8ba/beeswax/logs/ad_log.proto#L47